schematron/v2.2.0/tests/data/HVAC_example1.xml

```diff
@@ -130,15 +131,94 @@
                         </auc:CoolingSources>
                         <auc:Deliveries>
                             <auc:Delivery ID="Delivery-A">
+                                <auc:DeliveryType>
+                                    <auc:CentralAirDistribution>
+                                        <auc:AirDeliveryType>Central fan</auc:AirDeliveryType>
+                                        <auc:TerminalUnit>CAV terminal box no reheat</auc:TerminalUnit>
+                                        <auc:ReheatSource>Heating plant</auc:ReheatSource>
+                                        <auc:ReheatControlMethod>Single Maximum</auc:ReheatControlMethod>
+                                        <auc:ReheatPlantID IDref="HeatingPlant-A" />
+                                    </auc:CentralAirDistribution>
+                                </auc:DeliveryType>
+                                <auc:HeatingSourceID IDref="HeatingSource-A" />
+                                <auc:CoolingSourceID IDref="CoolingSource-A" />
                                 <auc:Capacity>123</auc:Capacity>
                                 <auc:CapacityUnits>gpm</auc:CapacityUnits>
                                 <auc:YearInstalled>2020</auc:YearInstalled>
+                                <auc:Quantity>1</auc:Quantity>
+                                <auc:DeliveryCondition>Good</auc:DeliveryCondition>
+                            </auc:Delivery>
+                            <auc:Delivery ID="Delivery-B">
+                                <auc:DeliveryType>
+                                    <auc:ZoneEquipment>
+                                        <auc:FanBased>
+                                            <auc:FanBasedDistributionType>
+                                                <auc:FanCoil>
+                                                    <auc:FanCoilType>Mini-split</auc:FanCoilType>
+                                                    <auc:HVACPipeConfiguration>1 pipe</auc:HVACPipeConfiguration>
+                                                </auc:FanCoil>
+                                            </auc:FanBasedDistributionType>
+                                        </auc:FanBased>
+                                    </auc:ZoneEquipment>
+                                </auc:DeliveryType>
+                                <auc:HeatingSourceID IDref="HeatingSource-A" />
```

Comment:
@nllong for `auc:DeliveryType/auc:ZoneEquipment` does it make sense to require both an `auc:HeatingSource` and `auc:CoolingSource`?  For example, we may have Chilled beam.  Do we want / need to enumerate these out and say something to the effect: "For `auc:ConvectionType=Chilled beam`, we expect a CoolingSourceID"...?

Reply:
Ideally we would create the enumeration links between the system and the heating/cooling source, but I don't think we should write a schematon test to handle this case. Ideally this first is implemented in our testsuite/documentation.

Reply:
Tracking in #44